norfair/tracker.py

```diff
@@ -13,18 +13,17 @@ def __init__(
         self,
         distance_function: Callable[["Detection", "TrackedObject"], float],
         distance_threshold: float,
-        hit_inertia_min: int = 10,
-        hit_inertia_max: int = 25,
+        hit_counter_max: int = 15,
         initialization_delay: Optional[int] = None,
+        pointwise_hit_counter_max: int = 5,
```

Comment:
According to my math this should be actually 4 instead of 5, to be the closest as it was when we used `point_transience`?